src/Compilers/CSharp/Test/Semantic/Semantics/ObjectAndCollectionInitializerTests.cs

```diff
@@ -3294,15 +3348,24 @@ static int Main()
             Initializer: 
               IObjectOrCollectionInitializerOperation (OperationKind.ObjectOrCollectionInitializer, Type: A) (Syntax: '{ 4, 5, 6 }')
                 Initializers(3):
-                    ICollectionElementInitializerOperation (AddMethod: void A.Add(System.Int32 i)) (IsDynamic: False) (OperationKind.CollectionElementInitializer, Type: System.Void, IsImplicit) (Syntax: '4')
+                    ICollectionElementInitializerOperation (AddMethod: void A.Add(System.Int32 i)) (OperationKind.CollectionElementInitializer, Type: System.Void, IsImplicit) (Syntax: '4')
```

Comment:
>void A.Add(System.Int32 i) [](start = 71, length = 26)

Do we have any tests covering "Add" extension methods. It would be good to have some. Especially for C#, it is interesting to see what happens with the implicit receiver.